frontend/src/App.tsx

```diff
@@ -94,12 +105,21 @@ function App() {
       onChangeTodo={onChangeTodo}
     ></TodoList>
   ) : !loading && todos.length === 0 && selectedFilter === Category.ALL ? (
-    <h3 className="msg-text">NO TODOS</h3>
+    <button onClick={onFocusInput} className="msg-box">
```

Comment:
Create a new component, called `Alert`, which is used to display the empty state for Todos and the error.

And once again, do not use the word "tasks".